lib/stores/DataStore.js

```diff
@@ -96,36 +60,25 @@ class DataStore extends EventEmitter {
      *
      * http://tus.io/protocols/resumable-upload.html#concatenation
      *
-     * @param  {object} req http.incomingMessage
+     * @param {object} stream stream.Readable
+     * @param {string} file_id Name of file
+     * @param {integer} offset starting offset
      * @return {Promise}
      */
-    write(req) {
-        log('[DataStore] write');
-        return new Promise((resolve, reject) => {
-            // Stub resolve for tests
-            const offset = 0;
-
-            this.emit(EVENTS.EVENT_UPLOAD_COMPLETE, { file: null });
-            return resolve(offset);
-        });
+    write(stream, file_id, offset) {
+        return Promise.resolve(0);
     }
 
     /**
      * Called in HEAD requests. This method should return the bytes
      * writen to the DataStore, for the client to know where to resume
      * the upload.
      *
-     * @param  {string} id     filename
-     * @return {Promise}       bytes written
+     * @param {string} file_id Name of file
+     * @return {Promise} bytes written
      */
-    getOffset(id) {
-        return new Promise((resolve, reject) => {
-            if (!id) {
-                return reject(ERRORS.FILE_NOT_FOUND);
-            }
-
-            return resolve({ size: 0, upload_length: 1 });
-        });
+    getOffset(file_id) {
+        return Promise.resolve({ size: 0, upload_length: 1 });
```

Comment:
Is there a reason for the non-zero upload length?

```suggestion
        return Promise.resolve({ size: 0, upload_length: 0 });
```